meta-resin-common/recipes-connectivity/networkmanager/files/99dhcp_ntp

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+added_servers_file=/tmp/chrony_added_dhcp_ntp_servers
+
+case "$2" in
+	dhcp4-change|up|connectivity-change)
+	if [[ -n $DHCP4_NTP_SERVERS ]]; then
+    # Tell chronyd to use this server
+		/usr/bin/chronyc add server $DHCP4_NTP_SERVERS
```

Comment:
What happens if `DHCP4_NTP_SERVERS` contains more than one server?

Reply:
looks like a :bug:

Reply:
This and indentation issues addressed in the recent commits